gprofiler/utils/__init__.py

```diff
@@ -273,7 +273,7 @@ def pgrep_exe(match: str) -> List[Process]:
         try:
             if pattern.match(process_exe(process)):
                 procs.append(process)
-        except psutil.NoSuchProcess:  # process might have died meanwhile
+        except (psutil.NoSuchProcess, MissingExePath):  # process might have died meanwhile
```

Comment:
`MissingExePath` is what we get for kernel threads, per the documentation of `process_exe`. We should not try to query them to begin with.

I think you started getting it now because `pgrep_exe` was unused for a while, and lately we added the `MissingExePath` exception.

The `except MissingExePath` is a bit confusing though, ideally we should skip kernel threads in this loop. Please see if it's possible with psutil API. Otherwise, you can add a comment on `MissingExePath` explaining that this is what you get for kernel threads, and we ignore them